packages/use-contractkit/src/screens/ledger.tsx

```diff
@@ -69,14 +74,15 @@ export const Ledger: React.FC<ConnectorProps> = ({
             <button
               className="tw-mt-6 tw-px-4 tw-py-2 tw-border tw-border-transparent tw-rounded-md tw-shadow-sm tw-text-base tw-font-medium tw-text-white tw-bg-gradient-to-r tw-from-purple-600 tw-to-indigo-600 hover:tw-from-purple-700 hover:tw-to-indigo-700"
               onClick={submit}
+              disabled={submitting}
             >
               {submitting ? (
-                <div className="tw-my-8 tw-flex tw-items-center tw-justify-center">
+                <div className="tw-flex tw-items-center tw-justify-center">
```

Comment:
Tinkered with these styles until the button stayed the same size while showing the loader